docs/source/markdown/options/userns.container.md

```diff
@@ -4,14 +4,14 @@
 ####> are applicable to all of those.
 #### **--userns**=*mode*
 
-Set the user namespace mode for the container. It defaults to the **PODMAN_USERNS** environment variable. An empty value ("") means user namespaces are disabled unless an explicit mapping is set with the **--uidmap** and **--gidmap** options.
+Set the user namespace mode for the container. It defaults to the **PODMAN_USERNS** environment variable.  When the environment variable is not set, it defaults to **host**, not to "".  An empty value ("") means user namespaces are disabled unless an explicit mapping is set with the **--uidmap** and **--gidmap** options.
```

Comment:
an empty string `""` or `host` should be equivalent so I don't think it is necessary to specify the `not to ""` part.

Reply:
> An empty value ("") means user namespaces are disabled unless an explicit mapping is set with the **--uidmap** and **--gidmap** options.

I think this is not correct.  When I call rootless podman without `--userns=`, for that user /etc/subuid contains `d:100000:65536`, and the container creates (from its perspective as root) a file, on the host the file is owned by UID 100000.  For me this means that `newuidmap` is utilized and therefore user-namespace is entered.

The cited sentence says in this case: user namespaces are enabled.  Isn’t this a contradiction?

Reply:
> The cited sentence says in this case: user namespaces are enabled.

READ

The cited sentence says in this case: user namespaces are DISABLED. Isn’t this a contradiction?

Reply:
rootless will always run in a user namespace, there is no way to make podman work without this
host or "" just means that there is no extra user namespace created for this container, instead it will use the one from the current process.